shared/codec/proto/codec_test.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package utility;
```

Comment:
Maybe one of these instead?
s/utility/shared?
s/utility/codec?
s/utility/proto?